src/components/common/Select/Select.tsx

```diff
@@ -1,58 +1,2 @@
-import { Select, SelectT } from '@terraware/web-components';
+import { Select } from '@terraware/web-components';
 export default Select;
-
-export type SelectItem = {
-  label: string;
-  value: string;
-};
-
-export interface SelectValueProps {
-  onChange: (newValue: string) => void;
-  label?: string;
-  disabled?: boolean;
-  id?: string;
-  className?: string;
-  helperText?: string | string[];
-  placeholder?: string;
-  errorText?: string | string[];
-  warningText?: string | string[];
-  selectedValue?: string;
-  readonly?: boolean;
-  options?: SelectItem[];
-  fullWidth?: boolean;
-  hideArrow?: boolean;
-  onBlur?: () => void;
-  onFocus?: () => void;
-  fixedMenu?: boolean;
-}
-
-/**
- * This is a simple component that takes in a tuple { label, value }
```

Comment:
I can update all usages including the existing PRs for checkin and withdrawal quantities - after it is merged to main.

Reply:
Ok, so I will not replace them now

Reply:
Replace them now, I will rename SelectValue to Dropdown when I merge my changes later.